operators/src/processing/point_in_polygon.rs

```diff
@@ -77,7 +76,7 @@ impl VectorOperator for PointInPolygonFilter {
     }
 }
 
-pub type InitializedPointInPolygonFilter = InitializedOperatorImpl<(), VectorResultDescriptor, ()>;
+pub type InitializedPointInPolygonFilter = InitializedOperatorImpl<VectorResultDescriptor, ()>;
```

Comment:
I guess we should use the moment to introduce an empty dummy struct here as state.